frontend/src/common/updatecheck.ts

```diff
@@ -0,0 +1,32 @@
+import AxiosStatic, { AxiosRequestConfig } from 'axios';
+
+const axiosBaseCfg: AxiosRequestConfig = {
+  headers: {
+    'Content-Type': 'application/json'
+  }
+};
+
+const axios = AxiosStatic.create(axiosBaseCfg);
+
+export class LatestVersionDto {
+  constructor(public stable: string, public beta: string) { }
+}
+
+class UpdatesService {
+  public async get(): Promise<LatestVersionDto> {
+    return axios.get('https://api.cryptomator.org/updates/hub.json')
```

Comment:
Shall we put the currently running version in the `User Agent` so we can use that for some statistical analysis?

Reply:
And maybe even a hash of the hub installation id?

Reply:
Since admin rights are currently needed to access the hubid, but the server info resides inside the settings section, we skip this enhancement for now.

Reply:
> currently running version in the User Agent 

Implemented in https://github.com/cryptomator/hub/pull/121/commits/0eaa14ac89c1bd27ee80805c3407844b81ab929b

Reply:
See https://github.com/cryptomator/hub/pull/121#discussion_r945896977